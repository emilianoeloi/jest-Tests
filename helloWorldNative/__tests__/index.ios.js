import 'react-native';
import React from 'react';
import Index from '../index.ios.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

const __DEV__ = false;

describe('index iOS', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Index />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('change Platform', () => {
    const component = renderer.create(
      <Index />
    );
    expect(component).toMatchSnapshot();

    let tree = component.toJSON();
    tree.props.onPress();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
