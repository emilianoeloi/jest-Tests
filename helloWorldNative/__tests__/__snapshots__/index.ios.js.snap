exports[`index iOS change Platform 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  onPress={[Function]}
  underlayColor="black">
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Click
  </Text>
</TouchableHighlight>
`;

exports[`index iOS change Platform 2`] = `
<TouchableHighlight
  activeOpacity={0.85}
  onPress={[Function]}
  underlayColor="black">
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Click
  </Text>
</TouchableHighlight>
`;

exports[`index iOS renders correctly 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  onPress={[Function]}
  underlayColor="black">
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Click
  </Text>
</TouchableHighlight>
`;
